### DISCUSSION TITLE
Swapped to JS UI and almost all devices stuck in ProtocolInfo

### DISCUSSION THREAD

**Discussion by Sevyn13** [zwave-js/zwave-js-ui#3864] [Q&A] [answered]
I followed the instructions [here](https://www.home-assistant.io/integrations/zwave_js/#how-do-i-switch-between-the-official-z-wave-js-add-on-and-the-z-wave-js-ui-add-on) to swap from Z-Wave JS to Z-Wave JS UI. After I was done almost all of my devices are unavailable and stuck in ProtocolInfo. I tried reinterviewing but it didn't help. Not sure what to do. Really don't want to remove them and start over.

**Comment by kpine** [comment, marked as answer]
Battery devices need to be manually woken to finish the interview.

This is a better guide: https://community.home-assistant.io/t/switching-z-wave-js-addons-with-minimal-downtime-z-wave-js-official-to-z-wave-js-ui-community/409904
